pkg/errors/errors.go

```diff
@@ -0,0 +1,35 @@
+package errors
+
+import (
+	"errors"
+	"fmt"
+)
+
+type Error struct {
+	Code    string
+	Message string
+	Err     error
+	Params  []interface{}
+}
```

Comment:
Thank you for refactoring 🙏🏻 

Now I think we could de-export these fields, since we should force interaction via methods.

Reply:
Nice point

Reply:
done